### DISCUSSION TITLE
Cannot install NI-DAQmx package

### DISCUSSION THREAD

**Discussion by jbscherrer** [bonsai-rx/bonsai#1127] [Q&A]
I'm migrating computers and trying to install the National Instruments DAQmx package. However, after installing the package, the nodes do not show up in the Bonsai toolbox. When I try to open a workflow with DAQmx nodes included, I get this error:

![image](https://user-images.githubusercontent.com/39060975/201207499-47c61071-55c2-4e12-bb48-e69cb68b9f1f.png)
[for search indexing: Could not load file or assembly 'NationalInstruments.DAQmx.dll' or one of its dependencies']

 This issue has been described in multiple previous posts, but none of the solutions are working for me. I've made sure the National Instruments .NET Framework support is installed, see below:

![image](https://user-images.githubusercontent.com/39060975/201207749-8e38dea3-028a-444f-9638-cfb41691563a.png)

I've also tried replacing the package folder in AppData with one from a working installation on a different computer, including `NationalInstruments.DAQmx.dll`. The problem persists in 32bit as well as 64bit applications.

Let me know if anyone has any ideas! Thanks!

**Comment by jbscherrer** [comment]
I've also tried uninstalling and reinstalling Bonsai and the NI-DAQmx application from National Instruments, to no effect.

**Reply by glopesdev** [reply]
This does looks like a missing indirect dependency, but I cannot seem to reproduce it on my side. Can you tell us which version of the DAQmx installer you are running, and also which version of Windows?

**Reply by glopesdev** [reply]
I would also suggest moving this to the DAQmx repo so we can keep a centralized place to track this down: https://github.com/bonsai-rx/daqmx/issues/3

**Reply by jbscherrer** [reply]
Sorry for the delay. Just added this to the thread in the DAQmx repo. Thanks!
